### ISSUE TITLE
breitbart.com

### ISSUE THREAD

**Issue by wanhose** [wanhose/cookie-dialog-monster#633]
#### Browser
Firefox (127.0)
#### Reason
Page contains visual glitches
#### URL
https://www.breitbart.com/politics/2024/07/02/joe-biden-blames-debate-performance-foreign-travel/
#### Version
7.0.4

**Comment by M1Aston** [comment]
As with another website I reported, without allowing social media elements in the cookie settings, all kinds of embedded stuff will not load at all.

**Comment by wanhose** [comment]
Hi @M1Aston,

Thanks for the information, I was able to find a workaround to show you the tweet content (not the embedded one) but at least you will be able to see the tweet message. Please tell me if it works for you after your next reboot.

Sorry for the delay, I'm very busy lately.

**Comment by M1Aston** [comment]
Yes, the text is there now, but everything else (video, image) is not.
Not ideal, but it's much better than before, so thanks for the update.

**Comment by M1Aston** [comment]
There is a problem. The comments won't load because of the cookie settings.
Click on the comments button below an article and see for yourself.

**Comment by M1Aston** [comment]
Another problem here: no video.
https://www.breitbart.com/clips/2024/07/07/tuberville-biden-is-sick-kamala-harris-probably-the-candidate-running-against-trump/

**Comment by wanhose** [comment]
Hi @M1Aston,

This case is very interesting. I found a workaround for each issue described here but the thing is due browser limitations, one of them won't work on Chrome.

Thanks for the feedback, I will keep you posted! 🙇‍♂️
